src/main/java/racingcar/model/Car.java

```diff
@@ -3,44 +3,43 @@
 import java.util.Objects;
 
 public final class Car {
-    private static final Distance INIT_STATE = Distance.ZERO;
-    private static final Distance MOVE_STATE = Distance.ONE;
+    private static final Distance INIT_DISTANCE = Distance.ZERO;
+    private static final Distance MOVE_DISTANCE = Distance.ONE;
+
     private final CarName name;
-    private final Distance state;
+    private final Distance distance;
 
     public Car(CarName name) {
-        validate(name);
-        this.name = name;
-        this.state = INIT_STATE;
+        this(name, INIT_DISTANCE);
     }
 
-    public Car(CarName name, Distance state) {
-        validate(name, state);
+    public Car(CarName name, Distance distance) {
+        validate(name, distance);
         this.name = name;
-        this.state = state;
+        this.distance = distance;
     }
 
-    private void validate(CarName name) {
+    private void validate(CarName name, Distance distance) {
         Objects.requireNonNull(name, "Car 생성시 전달된 carName 이 올바르지 않습니다. : name is null");
-    }
-
-    private void validate(CarName name, Distance state) {
-        Objects.requireNonNull(name, "Car 생성시 전달된 carName 이 올바르지 않습니다. : name is null");
-        Objects.requireNonNull(state, "Car 생성시 전달된 state 가 올바르지 않습니다. : state is null");
+        Objects.requireNonNull(distance, "Car 생성시 전달된 distance 가 올바르지 않습니다. : distance is null");
     }
 
     public Car move(MovingStrategy movingStrategy) {
         if (movingStrategy.movable()) {
-            return new Car(name, Distance.add(state, MOVE_STATE));
+            return new Car(name, Distance.add(distance, MOVE_DISTANCE));
         }
-        return new Car(name, new Distance(state.value()));
+        return new Car(name, new Distance(distance.value()));
+    }
+
+    public boolean isSameDistance(Distance distance) {
+        return distance.equals(distance);
```

Comment:
파라미터 자신을 비교하고 있습니다. 무조건 true가 되겠네요.
```suggestion
        return this.distance.equals(distance);
```
이 경우에는 false가 되는 테스트 케이스를 작성하시면 사전에 방지할 수 있겠네요. 😄

Reply:
앗!! ㅠㅠ this를 빼먹었네요 !! 😢
결과가 true/false로 나올 수 있는 조건에 대해서 두가지 경우를 모두 테스트해보면서 구현이 올바르게 됐는지 잘 확인해봐야겠습니다!! 감사합니다 ㅠㅠ